simd/simd-memtest/driver.c

```diff
@@ -40,6 +32,10 @@ int main(int argc, char* argv[])
     printf("number of warmups       = %zu\n", nwarm);
 
     size_t bytes = nelem * sizeof(double);
+    // the number of bytes actually allocated are padded out by 7 * max_stride
+    // to provide a buffer for non-power of true stides which read slightly beyond
+    // the end of the array
+    size_t alloc_bytes = (nelem + 7 * 64) * sizeof(double);
```

Comment:
The program was crashing for me because of the loop condition `i<n` in the strided loops, which isn't correct for many strides (reads/writes beyond the end of the allocated area).